/**:
  ros__parameters:
    convolution_padding: 10
    max_edges_per_block: 20
    radian_threshold: 2.0
    distance_diff_threshold: 0.3
    range_ratio_threshold: 0.02
    edge_threshold: 0.8
    surface_threshold: 0.02
    min_range: 0.1
    max_range: 100.0
